test/unit/mock/files/hosts.allow

```diff
@@ -0,0 +1,17 @@
+#
+# hosts.allow   This file contains access rules which are used to
+#               allow or deny connections to network services that
+#               either use the tcp_wrappers library or that have been
+#               started through a tcp_wrappers-enabled xinetd.
+#
+#               See 'man 5 hosts_options' and 'man 5 hosts_access'
+#               for information on rule syntax.
+#               See 'man tcpd' for information on tcp_wrappers
+#
+# LOCALHOST (ALL TRAFFIC ALLOWED) DO NOT REMOVE FOLLOWING LINE
+ALL: 127.0.0.1, [::1]
+# Allow SSH (you can limit this further using IP addresses - e.g. 192.168.0.*)
+sshd: ALL
+# Added for testing
+LOCAL : [fe80::]/10 : deny
+UNKNOWN, sshd : 127.0.1.154, [:fff:fAb0::] : deny : /etc/bin/
```

Comment:
Yeah, that's an option, but it requires the user to understand the resource's technical implementation. It doesn't really help them answer the question.

Think about what tcpwrappers is for... it controls access to daemons. A daemon is the primary key. A user shouldn't have to worry about a line protecting multiple daemons -- they want to ask a question about a daemon.

I think the right solution is, while parsing the lines, if there are multiple daemons listed on a line, add multiple entries to the hash used by the FilterTable.